js/listing.js

```diff
@@ -1,16 +0,0 @@
-(async () => {
```

Comment:
Removing this will break the extension repos as they all use this JS file.

Reply:
Ahh, yup. for the schema's right?

Reply:
Yes.

Reply:
Where does this script get called?
The schema sites are being published by this action in each repo.
https://github.com/stac-extensions/template/blob/main/.github/workflows/publish.yaml

Reply:
The index.html is directly in the gh-pages branch for each repo, e.g. https://github.com/stac-extensions/version/blob/gh-pages/index.html

Reply:
ahhh Thanks.